perceval/backend.py

```diff
@@ -192,6 +246,18 @@ def _init_client(self, from_archive=False):
         raise NotImplementedError
 
 
+def _remove_key_from_nested_dict(nested_dict, path_to_field):
```

Comment:
Although it could be a corner case, items with a high nesting level could cause problems in terms of memory consumption. If possible, it would be better to use a loop.

Reply:
I think this is one of the cases where recursion works better. Nested fields shouldn't be longer than 4 or 5 keys? I tested with 100 and 1000 nested keys and it works without problems. If wants to filter something with a deep of 10 keys...it's probably because he's doing something terrible wrong.